<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airbus Fly-By-Wire Development</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabConnection">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>CONNECT</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxConfigurationIndex">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>10</y>
          <width>42</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabPitch">
       <attribute name="title">
        <string>Pitch</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="ActualDemandChart" name="pitchRate"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelPitch"/>
        </item>
        <item row="1" column="0">
         <widget class="ActualDemandChart" name="c_star"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRoll">
       <attribute name="title">
        <string>Roll</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="ActualDemandChart" name="rollRate"/>
        </item>
        <item row="2" column="0">
         <widget class="ActualDemandChart" name="bank"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelRoll"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActualDemandChart</class>
   <extends>QGraphicsView</extends>
   <header>ActualDemandChart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
